FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -135,7 +253,16 @@ public static function configDataProvider()
                 'findologicDI' => false,
                 'isSearchPage' => false,
                 'isCategoryPage' => true,
-                'expected' => false
+                'expected' => true
+            ],
+            'FINDOLOGIC is active on category pages but it was disabled in the plugin config' => [
+                'ActivateFindologic' => true,
+                'ShopKey' => '0000000000000000ZZZZZZZZZZZZZZZZ',
+                'ActivateFindologicForCategoryPages' => true,
+                'findologicDI' => false,
+                'isSearchPage' => false,
+                'isCategoryPage' => true,
+                'expected' => true
             ]
```

Comment:
I am missing the test case `In any other case where none of these apply => should return false`.

Reply:
This case is also already covered in one of the cases above.